### PR TITLE
Added default action update flag in TableEntry.

### PR DESCRIPTION
**Pull request by samar-abdi**

### CODE REVIEW

**Comment by samar-abdi** [comment]
Ugh. Sorry!

**Comment by samar-abdi** [comment]
Incorporated Antonin's and Tom's feedback. PTAL.

File: proto/docs/p4runtime.md [hunk 1]
```diff
@@ -126,7 +142,22 @@ table_entry {
 
 If `match` for a field is not defined, then the field is treated as don't care
 when constructing the match key for the table entry. Note that leaving match field
-undefined is only allowed for ternary and LPM fields.
+undefined is only allowed for ternary and LPM fields. The following table entry
```
Comment (teverman) [inline proto/docs/p4runtime.md]: The old code allowed for the possibility of don't care range matches - see lines 159-160 of the old code.  Are we back to not allowing don't care range matches?

File: proto/docs/p4runtime.md [hunk 2]
```diff
@@ -108,15 +108,31 @@ format):
...
-of a table:
+to true when it wants to change the default action within a table. The following
+table illustrates this use case. The ids in the comments refer to the P4 compiler
+assigned ids to the entities. We assume that both actions `bar` and `baz` do not
+take any arguments.
 
 ```
+table foo {             // table id: 1
+  key = {
+    hdr.foo : ternary;  // field id: 10
```
Comment (teverman) [inline proto/docs/p4runtime.md]: Missing closing } for key.

File: proto/docs/p4runtime.md (hunk 1 above)
Comment (antoninbas) [reply proto/docs/p4runtime.md]: I think that's just an omission

**Comment by antoninbas** [comment]
@samar-abdi @teverman I'll merge this as soon as the regression passes

**Pull request merged**